src/main/java/seedu/address/logic/commands/RemoveCommand.java

```diff
@@ -102,11 +102,6 @@ public boolean equals(Object other) {
             return true;
         }
 
-        // instanceof handles nulls
-        if (!(other instanceof EditCommand)) {
```

Comment:
shouldn't this be replaced by `if (!(other instanceof RemoveCommand))` instead

Reply:
Made the necessary changes including the test case